NEWS.md

```diff
@@ -212,6 +212,9 @@ Library improvements
 
   * `notify` now returns a count of tasks woken up ([#19841]).
 
+  * Introduce nonstandard string literal `raw"..."` for creating strings
```

Comment:
Maybe "New" instead of "Introduce". At least that what is seems like is being used in many of the other bullets for new functions and macros.